src/Microsoft.ComponentDetection.Detectors/pip/PythonResolver.cs

```diff
@@ -210,10 +214,57 @@ private void AddGraphNode(PythonResolverState state, PipGraphNode parent, string
         }
         else
         {
-            var node = new PipGraphNode(new PipComponent(name, version));
+            var node = new PipGraphNode(new PipComponent(name, version, license: license, author: author));
             state.NodeReferences[name] = node;
             parent.Children.Add(node);
             node.Parents.Add(parent);
         }
     }
+
+    private string GetAuthorFromProject(PythonProject project)
+    {
+        if (!string.IsNullOrEmpty(project.Info?.Author))
+        {
+            return project.Info.Author;
+        }
+
+        if (!string.IsNullOrEmpty(project.Info?.Maintainer))
+        {
+            return project.Info.Maintainer;
+        }
+
+        if (!string.IsNullOrEmpty(project.Info?.AuthorEmail))
+        {
+            return project.Info.AuthorEmail;
+        }
+
+        if (!string.IsNullOrEmpty(project.Info?.MaintainerEmail))
+        {
+            return project.Info.MaintainerEmail;
+        }
+
+        // If none of the fields are populated, return null.
+        return null;
+    }
+
+    private string GetLicenseFromProject(PythonProject project)
+    {
+        if (project.Info?.Classifiers != null)
+        {
+            var licenseClassifiers = project.Info.Classifiers.Where(x => x.StartsWith("License ::")).ToList();
+
+            // Split the license classifiers by the " :: " and take the last part of the string
+            licenseClassifiers = licenseClassifiers.Select(x => x.Split(" :: ").Last()).ToList();
```

Comment:
is the split always like that ` :: `? If so could you link to Python doc in PR description in case there is an update done upstream that we need to include here too. Additionally, a constant for ` :: ` divider string and using it here and in line 254 would help for any future refactoring.

Reply:
Updated the PR description with the relevant docs and added the constants.